nautobot_chatops_ansible/worker.py

```diff
@@ -88,16 +91,16 @@ def get_inventory(dispatcher, inventory, group):
 
     if not group:
         data = tower.get_tower_inventory_groups(inventory=inventory)
+        LOGGER.debug("Data result: %s", data)
         dispatcher.prompt_from_menu(
-            f"ansible get-inventory {inventory}",
+            f'ansible get-inventory "{inventory}"',
```

Comment:
```suggestion
            f"ansible get-inventory '{inventory}'",
```
To keep the quotations consistent

Reply:
+1 to this suggestion. Especially with Slack and handling Double Quotes.

Reply:
After re-reading this, I realize you're passing a string that might require double quotes in it when being passed, as the double quotes aren't used to render `{inventory}`

I think this suggestion may not be necessary, unless you do something like (which to me is a little silly):

`"ansible get-inventory \"{inventory}\""`

Summary: So if the double-quotes are required for dispatcher.prompt_from_menu(), I'd say ignore this suggestion.  Otherwise use it.

Reply:
I would follow  the docs from the chatops plugin on how to quote strings.  In this case, I would use the suggested change.  https://github.com/nautobot/nautobot-plugin-chatops/blob/develop/docs/design.md#multi-word-parameters